neuromation/api/blob_storage.py

```diff
@@ -51,11 +51,10 @@
 
 
 def _format_bucket_uri(bucket_name: str, key: str = "") -> URL:
-    return URL.build(scheme="blob", host=bucket_name, path="/" + key.lstrip("/"))
-
-
-def _extract_key(uri: URL) -> str:
-    return uri.path.lstrip("/")
+    if key:
+        return URL(f"blob:{bucket_name}/{key}")
```

Comment:
Not sure is this is relevant, but just for the case if `bucket_name` or `key` contain something like `%20` I would use `URL.build()`.

Reply:
I was not successful with URL.build producing a relative URL =(. I do think it's a good idea.
```python
>>> URL.build(scheme="blob", path="folder/key.txt")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/taras/.pyenv/versions/3.7.4/lib/python3.7/site-packages/yarl/__init__.py", line 202, in build
    raise ValueError('Can\'t build URL with "scheme" but without "host".')
ValueError: Can't build URL with "scheme" but without "host".
```
I will try to operate on full URLs. Give me a sec.

Reply:
Hmm, this seem to work:
```python
>>> yarl.URL("blob:") / "foo" / "мрія"
URL('blob:foo/%D0%BC%D1%80%D1%96%D1%8F')
```

Reply:
Hmm, strangely the %20 case does not actually work:
```python
>>> yarl.URL.build(scheme="blob", host="default", path="/foo/%20object1")
URL('blob://default/foo/%20object1')
>>> yarl.URL.build(scheme="blob", host="default", path="/foo/ object1")
URL('blob://default/foo/%20object1')
```
It does not requote the first one.

Reply:
It may be a bug in `yarl`.

I like `URL("blob:") / ...`.